### ISSUE TITLE
[Terra Form Validation] Background color of the select (with required error) is not matched with the input backround

### ISSUE THREAD

**Issue by KunigundaLauly** [cerner/terra-framework#1557]
# Bug Report

## Description

After submitting a form, the select bacgkround color should match with the input one. (white on default theme)

## Steps to Reproduce

1. Open the following link: https://engineering.cerner.com/terra-ui/components/terra-form-validation/form-validation/validation-events
2. Set the background to #F7ECFA
3. Press the Submit button
4. The input field background is white.
5. The select background color is transparent (not white).

## Additional Context / Screenshots

![image](https://user-images.githubusercontent.com/47626274/164452779-727960a7-bb6e-41ec-8132-ead9137ad52a.png)

## Expected Behavior

The form validated selects background should be the same as the input ones.

## Environment

* Component Name and Version: terra-form-validation 1.69.6
* Browser Name and Version: Chrome newest
* Node/npm Version: [e.g. Node 8/npm 5] -
* Webpack Version: -
* Operating System and version (desktop or mobile): windows 10

**Comment by chrismichalewicz** [comment]
UXPLATFORM-6598 has been created for this.

**Comment by KunigundaLauly** [comment]
up.

**Comment by chrismichalewicz** [comment]
@KunigundaLauly  We have an internal Jira logged for this so even if this github issue gets closed for being stale we will still include this request within our list of projects to prioritize for future work.

**Comment by KunigundaLauly** [comment]
@chrismichalewicz thank you! Will we get somewhere notified when this ticket is resolved & published?
